server/src/main/java/org/apache/iotdb/db/qp/constant/TqlParserConstant.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
use /*  rather than /**

Reply:
I replaced /** with /*. Thanks for your suggestion.